src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -233,6 +292,11 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
                 )}
               </ButtonConfirmed>
             )}
+            {isAlreadyApproved && (
+              <UnderlineButton disabled={approving || !isAlreadyApproved} onClick={handleRevokeApproval}>
+                Revoke approval
```

Comment:
Should we add a `(Test only)` if we want to merge this?

Reply:
I'll remove this if we merge